### ISSUE TITLE
Table component aria-sort attribute

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1191]
For column headers that are sortable, `aria-sort` should be added to the `<th>` elements. Currently, the examples of the table component only include `aria-sort` on the currently sorted column, but not the other column headers that can be sorted on.

For the column that is currently sorted, the `<th>` should include either `aria-sort="ascending"` or `aria-sort="descending"`.

For the columns that could be sorted but aren't the current sort value (e.g. include a `<button>` element instead of just text), the `<th>` element should include `aria-sort="none"`.

Examples showing this are available in the [wai-aria APT](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html) for Grid and [Deque](https://dequeuniversity.com/library/aria/tables/sf-sortable-grid).

**Comment by mcoker** [comment]
addressed in https://github.com/patternfly/patternfly-next/pull/1067
